style/pi-hole.css

```diff
@@ -575,7 +575,7 @@ td.details-control {
 }
 
 .form-control-feedback {
-  right: 0.5em;
+  right: 1em;
   width: 16px;
   height: 100%;
 }
```

Comment:
Remove this block (sorry... github interface won't let me suggest using the code).